src/qt/bitcoingui.h

```diff
@@ -187,6 +188,11 @@ public slots:
 #ifdef ENABLE_WALLET
     void setStakingStatus();
     void setAutoMintStatus();
+/** Set the hd-enabled status as shown in the UI.
+     @param[in] status            current hd enabled status
+     @see WalletModel::EncryptionStatus
+     */
+    void setHDStatus(int hdEnabled);
```

Comment:
nope,there isn't one,although it is what is being used on dash and most of the other dash forks,can be changed if needed.

Reply:
In my humble opinion; given the issue with the comment for the routine.  It's apparent the original developer copied the setEncryptionStatus comment and declaration; and only changed the first line of the comment (and the routine name).  Which looks to me like sloppy coding.

With that evidence presented; I think it would behoove PIVX to properly define it as a bool.

Especially given the use with `hdEnabledStatsChanged()`, which should also be changed to a bool also.

Of course, I don't feel I have authority to 'change request', and this is just a comment.  We can ask the pivx core devs their opinion.

Reply:
I agree with @CaveSpectre11.
However, I think there's no need to focus on the qt files right now, as the GUI is being completely reworked (https://github.com/PIVX-Project/PIVX/pull/954).
It's probably best to either
- rebase this PR on top of PR-954 and add the HD controls in the new interface.
or
- remove the GUI/QT edits from this PR and add them in a separate PR after PR-954 is merged.

Reply:
ill remove the qt code and push it,will add the qt ui needed for the hd status to new ui once new ui code is merged

Reply:
don't worry about the UI, i already have all of that designed and partially coded. Once this PR gets  fully reviewed and merged, only will need to code a good interface methods connected to this new functionality and done.